Pages/Index.cshtml.cs

```diff
@@ -1,47 +1,71 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.IO;
+using System.Net;
+
 
 namespace DotnetBunnyLogBrowser.Pages
 {
-	public class IndexModel : PageModel
+    public class IndexModel : PageModel

```

Comment:
Be careful here, this used tabs for indentation, and now you're mixing it with spaces. This will get you drawn and quartered (first time figuratively, second time literally) in the right audience. Luckily, we're not so high strung here. :)